docs/source/notes/autograd.rst

```diff
@@ -93,26 +93,25 @@ under heavy memory pressure, you might never need to use them.
 
 There are two main reasons that limit the applicability of in-place operations:
 
-1. Overwriting values required to compute gradients. This is why variables don't
```

Comment:
I don't know how much of this is true anymore. `log_` works:
![image](https://user-images.githubusercontent.com/5652049/39001837-392f816e-43c5-11e8-8f14-1c29a9f12efe.png) so I don't know in what cases values are overwritten